server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/mapper/DefaultJdbcRowMapper.java

```diff
@@ -61,6 +61,13 @@ public DefaultJdbcRowMapper(@NonNull ConnectionSession session) {
                     ConnectionSessionUtil.getNlsTimestampFormat(session)));
             mapperList.add(new OracleNumberMapper());
             mapperList.add(new OracleBinaryNumberMapper());
+        } else if (Objects.nonNull(dialectType) && dialectType.isDoris()) {
+            mapperList.add(new MySQLBitMapper());
+            mapperList.add(new MySQLDatetimeMapper());
+            mapperList.add(new MySQLYearMapper());
+            mapperList.add(new MySQLTimestampMapper());
```

Comment:
Thanks for your contributions! After a careful review, we've noticed that this section of the code appears to share similarities with an existing conditional statement : 
`if (Objects.nonNull(dialectType) && dialectType.isMysql()).`
 To maintain the code's efficiency and clarity, we suggest that consolidate these conditions into a single unified branch like ：
`if (Objects.nonNull(dialectType) && (dialectType.isMysql()) || dialectType.isDoris())`. 
If other codes have similar situations, you can consider modifying them together～

Reply:
Doris have owner column types, like boolean, HyperLogLog, STRUCT......